pkgs/stdenv/generic/default.nix

```diff
@@ -135,6 +135,8 @@ let
       inherit overrides;
 
       inherit cc;
+
+      isCross = targetPlatform != buildPlatform;
```

Comment:
@Ericson2314 FYI

Reply:
@Ericson2314 I'm really not a huge fan of making things more verbose and annoying for this kind of case... If we really need to have a more stringent review in these cases, maybe we can have Ofborg ping some nixpkgs-cross-committers group when `stdenv.isCross` is added to some file? @grahamc ?